test/opt/ir_loader_test.cpp

```diff
@@ -21,13 +21,16 @@
 
 #include "gtest/gtest.h"
 #include "source/opt/build_module.h"
+#include "source/opt/def_use_manager.h"
 #include "source/opt/ir_context.h"
 #include "spirv-tools/libspirv.hpp"
 
 namespace spvtools {
 namespace opt {
 namespace {
 
+const uint32_t kOpLineOperandLineIndex = 1;
```

Comment:
nit: contexpr?

Reply:
Done. It's weird that `constexpr` looks better than `const` for similar constants but spirv-tools does not use it. We intentionally do not use it for spirv-tools.

Reply:
For a long time we supported VisualStudio 2013 which did not support constexpr.